src/osemgrep/cli_login/Login_subcommand.ml

```diff
@@ -96,7 +96,7 @@ let run (conf : Login_CLI.conf) : Exit_code.t =
                     {|{"token_request_key": "|} ^ Uuidm.to_string session_id
                     ^ {|"}|}
                   in
-                  match Http_helpers.post ~body url with
+                  match Http.post ~body url with
```

Comment:
Might be better to restore Http_helpers_piaf.ml or something.

Reply:
I'll just restore it to `Http_helpers`